templates/web/base/around/on_map_list_items.html

```diff
@@ -14,7 +14,11 @@
     </li>
 [% ELSE %]
     <li class="item-list__item item-list__item--empty">
-        <p>[% loc('There are no reports to show.') %]</p>
+        [% IF js %]
```

Comment:
This is always true (because of the `PROCESS` sidebar.html which doesn't appear to use the variable), whether JS is being used or not, so will always be shown. The `js` variable is for displaying the new report form (hidden) on the around page; the thing you're wanting to test here is whether the URI has js=1 in it or not, so `c.get_param('js')`.